py-polars/docs/source/reference/expressions/functions.rst

```diff
@@ -97,7 +97,6 @@ These functions are available from the Polars module root and can be used as exp
    Expr.any
    Expr.approx_n_unique
    Expr.count
-   Expr.exclude
```

Comment:
`Expr.exclude` also exists under the `Columns / names` category, so this was a duplicate doc entry here.